src/core_modules/capture-ui/IconButton/IconButton.component.js

```diff
@@ -1,10 +1,36 @@
 // @flow
 import React, { useCallback } from 'react';
 import cx from 'classnames';
-import classes from './iconButton.module.css';
+import { withStyles } from '@material-ui/core/styles';
+import { colors } from '@dhis2/ui';
 import type { Props } from './iconButton.types';
 
-export const IconButton = ({ children, className, dataTest, onClick, ...passOnProps }: Props) => {
+const styles = {
+    button: {
+        cursor: 'pointer',
+        borderRadius: '3px',
```

Comment:
To maintain the current round border can we use `border-radius: 50%;` instead?

Reply:
Good point, but it was decided that these should look like the buttons provided from UI and the already existing "IconButtons" in our widgets. Is that ok or should we mention it in the design-meeting? 😊

Reply:
Sounds good! If you add a comment in the ticket about this decision should be enough. Thanks!